hadoop-hdds/common/src/main/resources/network-topology-default.yaml

```diff
@@ -30,32 +30,26 @@ prefix: /
 type: ROOT
 
 # Layer name
-defaultName: root
+defaultName: datacenter
 
 # Sub layer
 # The sub layer property defines as a list which can reflect a node tree, though
 # in schema template it always has only one child.
 sublayer:
   -
     cost: 1
-    prefix: dc
-    defaultName: datacenter
+    prefix: rack
+    defaultName: rack
     type: INNER_NODE
     sublayer:
-      -
-        cost: 1
-        prefix: rack
-        defaultName: rack
-        type: INNER_NODE
-        sublayer:
+        -
+          cost: 1
+          prefix: ng
+          defaultName: nodegroup
```

Comment:
There is no nodegroup in default topology.  We can remove this layer. 

Could you add a unit test in TestYamlSchemaLoader.java?  load this default topology configuration file, and check there is  3 level NodeSchema in NodeSchemaManager?

Reply:
@ChenSammi Thanks for the review. Updated the PR, please have a check.